Repair fstab::augeas for ensure absent.

qha opened the pull request:
Define fstab_match_line in fstab::augeas too as it is used if
ensure is absent.

Tune smoke tests to confirm.

This closes issue #9.

docwhat commented:
If you add "This closes issue #9" to your git commit message, it'll close #9 automatically as well. :-)

qha commented:
Right, thanks!

larstobi commented:
Thanks, @jdtcdr, that fixed ensure => absent for us, along with #12.

masimplo commented:
Thanks @qha

qha committed:
Repair fstab::augeas for ensure absent.

Define fstab_match_line in fstab::augeas too as it is used if
ensure is absent.

qha committed:
fstab: tune/add resources in/to test manifest

This provides coverage of ensure absent cases.

This closes issue #17.

qha commented:
Closed in favour of pr #15.
